### DISCUSSION TITLE
SSL error when self hosting behind caddy reverse proxy

### DISCUSSION THREAD

**Discussion by nabeelmoeen** [ente-io/ente#3652] [Q&A]
Hi,
I setup a self hosted instance of ente using the precompiled docker imagejmage from ghcr.io as per the docs.
Since port 8080 on the server was already in use I mapped the museum port 8080 to host 8081.
When testing curl localhost:8081 I get the expected result.

But when I access using the domain URL setup in the Caddy file I get the following error:

`curl: (35) error:14094438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error`

From the logs it doesn't appear the call is registered on the server
E.g. when using local host I can see the client IP and request registered in the log

**Comment by nabeelmoeen** [comment]
sorry, turns out i had a typo in the caddy config.

**Comment by A4alli** [comment]
Please share your setup. I wana try

**Reply by mngshm** [reply]
https://help.ente.io/self-hosting/guides/web-app 

an in-development documentation, but it works :p

**Reply by A4alli** [reply]
just run immich with Tailscale and serve without exposing any Ports. if you Wana try: https://github.com/immich-app/immich/discussions/14868
